apps/web/src/components/sections/testimonials/modern/testimonials.tsx

```
@@ -103,8 +103,8 @@ function Testimonials() {
             </CarouselContent>
           </Carousel>
 
-          <div className="dark:from-background pointer-events-none absolute inset-y-0 left-0 w-1/4 bg-linear-to-r from-white"></div>
-          <div className="dark:from-background pointer-events-none absolute inset-y-0 right-0 w-1/4 bg-linear-to-l from-white"></div>
+          <div className="md:dark:from-background pointer-events-none absolute inset-y-0 left-0 w-1/4 md:bg-linear-to-r md:from-white"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)